contractcourt/channel_arbitrator.go

```diff
@@ -455,31 +467,22 @@ func (c *ChannelArbitrator) relaunchResolvers() error {
 	// to prevent a db migration. We use all available htlc sets here in
 	// order to ensure we have complete coverage.
 	htlcMap := make(map[wire.OutPoint]*channeldb.HTLC)
-	for _, htlcs := range c.activeHTLCs {
```

Comment:
is `activeHTLCs` still used?

Reply:
yes, when the channel_arbitrator is in `StateDefault` and watching for expiring htlcs, `activeHTLCs` tracks what is happening in the link.